500 when get commits info of submodule

**Describe the bug**

Gogs fails to display repo in web interface and raises 500 error. This happens when gogs fails to fetch commits info from git submodules in bare repo. Gogs worked correctly two of three versions back on debian 10.

**Gogs version or commit**

Gogs: Go Git Service 0.12.0+dev

**Git version**

git version 2.20.1

**Operating system**

Debian GNU/Linux 10 (buster)

**Additional context**

```
2020/03/23 11:24:57 [ERROR] [...gogs/internal/route/repo/view.go:278 Home()] get commits info: get submodule "toolchain": revision does not exist
2020/03/23 11:24:57 [TRACE] Template: status/500
```

--- a/go.mod
+++ b/go.mod
@@ -18,7 +18,7 @@ require (
 	github.com/go-sql-driver/mysql v1.4.1
 	github.com/gogs/chardet v0.0.0-20150115103509-2404f7772561
 	github.com/gogs/cron v0.0.0-20171120032916-9f6c956d3e14
-	github.com/gogs/git-module v1.0.0
+	github.com/gogs/git-module v1.0.1
 	github.com/gogs/go-gogs-client v0.0.0-20200128182646-c69cb7680fd4
 	github.com/gogs/go-libravatar v0.0.0-20191106065024-33a75213d0a0
 	github.com/gogs/minwinsvc v0.0.0-20170301035411-95be6356811a

